recipes-support/salt/salt_3000.2.bb

```diff
@@ -55,6 +55,37 @@ PACKAGES += "\
            ${PN}-bash-completion \
 "
 
+RDEPENDS_${PN}-minion_append += "\
+    python3-aiodns \
+    python3-aiohttp \
+    python3-avahi \
+    python3-mmap \
+    python3-pyinotify \
+    python3-pyroute2 \
+    python3-pika \
+    python3-psutil \
+"
+
+# these dependencies should NOT be added to the salt bb recipe as they're added
+# here in the bbappend for the NIFeeds ni-skyline packages. In the future these
+# will be removed and skyline packages made to depend on them directly.
```

Comment:
Do the SL folks have any more context on this effort? Do we still need to add these?

Reply:
Yes, still needed. I asked them and they hadn't made the required changes to the skyline packages.